custom/rch/models.py

```diff
@@ -0,0 +1,159 @@
+import datetime
+from django.db import models
+from django.conf import settings
+from custom.rch.utils import (
+    fetch_beneficiaries_records,
+    find_matching_cas_record_id,
+)
+from jsonfield.fields import JSONField
+from custom.rch.const import (
+    MOTHER_RECORD_TYPE,
+    CHILD_RECORD_TYPE,
+    STATE_DISTRICT_MAPPING,
+    RCH_RECORD_TYPE_MAPPING,
+)
+
+
+class RCHRecord(models.Model):
```

Comment:
Seems like there is quite a lot of forking between `mother` and `child`. Did you consider using separate models or [model inheritance](https://docs.djangoproject.com/en/1.11/topics/db/models/#model-inheritance)?

Reply:
I started with that path but then ended up merging them as a single entity since that made it easier. Reverting it back to two different models.